guides/v2.3/extension-dev-guide/declarative-schema/index.md

```diff
@@ -30,7 +30,7 @@ Once a module is converted to the declarative schema approach, it cannot be reve
 
 * *Data patch* - A class that contains data modification instructions. It can have dependencies on other data or schema patches.
 
-* *Revertable data patch* - A patch that can be reverted as a module or patch is uninstalled or deleted. Revertable operations are Data Query Language (DQL) and Data Manipulation Language (DML) operations: INSERT, UPDATE.
+* *Revertable data patch* - A `data patch` that contains a `revert()` method (see [Reverting data patches]({{ page.baseurl }}/extension-dev-guide/declarative-schema/data-patches.html/#revertingDataPatches)) with provided revertable operations.
```

Comment:
We need to reconsider the meaning of "revertable operations" here. The current change says that the `revert()` method contains **revertable** operations. But in fact the `revert()` method contains **revert** operations. 
**Revertable** operations - operations that can be reverted. So it's not 100% correct to say that `revert()` method contains operations that can be reverted. It contains operations that revert revertable changes caused by the module. 
Also, I would provide some infomation about what changes are revertable (changes in the database) and what are not (changes in the core code).

Reply:
Thank you for your notice, @rogyar !

I've implemented your suggestions. Could you please verify a new solution?

Thank you!